### ISSUE TITLE
LaMa Loss Functions

### ISSUE THREAD

**Issue by amindehnavi** [lorenzo-stacchio/stable-diffusion_custom_inpaint#14]
Hi @lorenzo-stacchio,

Thank you for your work and for providing the training scripts.

Have you tested implementing the loss functions of the LaMa models? Do you have any ideas on whether using these losses, such as adversarial loss, could help improve the output images?
